src/main/groovy/com/jetbrains/python/envs/PythonEnvsPlugin.groovy

```diff
@@ -83,51 +83,118 @@ class PythonEnvsPlugin implements Plugin<Project> {
                     }
                 }
 
-                condaInstall(project, null, condaExecutable, envs.basePackages)
-//            doFirst {
-//                if (!myExt.bootstrapDirectory.exists()) {
-//                    myExt.bootstrapDirectory.mkdir()
-//                }
-//            }
+                condaInstall(project, condaExecutable, envs.condaBasePackages)
             }
         }
     }
 
-    def createBootstrapJython(project, jythonBootstrapDir) {
-        project.tasks.create(name: 'bootstrapJython') {
-            def conf = project.configurations.jython
+    private void installPythonBuild(Project project, File installDir) {
+        project.tasks.create(name: 'install_python_build') {
+            outputs.dir(installDir)
+            onlyIf {
+                !installDir.exists()
+            }
+
+            doLast {
+                File pythonBuildZip = new File(project.buildDir, "python-build.zip")
+                getClass().getClassLoader().getResource('python-build.zip').withInputStream { is ->
+                    pythonBuildZip.withOutputStream { os ->
+                        os << is
+                    }
+                }
+
+                File temporaryUnzipFolder = new File(project.buildDir, "python-build-tmp")
+                project.ant.unzip(src: pythonBuildZip, dest: temporaryUnzipFolder)
```

Comment:
It will be shorter to use gradle task Copy for that: from(zipTree(....)) into ...